Client/Game.cpp

```diff
@@ -86,9 +87,16 @@ Game::Game() {
 		gsn->deserialize(buffer);
 		/*TODO: graphics update based on the game state*/
```

Comment:
New Issue we can start?

Reply:
The TODO can be removed since it is resolved with this PR.